hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -211,6 +211,10 @@ function ozonecmd_case
       OZONE_CLASSNAME=org.apache.hadoop.ozone.debug.OzoneDebug
       OZONE_RUN_ARTIFACT_NAME="ozone-tools"
     ;;
+    jmxget)
+      OZONE_CLASSNAME='org.apache.hadoop.hdfs.tools.JMXGet'
```

Comment:
I'm not sure we can simply use `JMXGet` from HDFS, as its help still references NameNode:

```
$ ozone jmxget -help
usage: jmxget options are:
 -help                           print help
 -localVM <VM's connector url>   connect to the VM on the same machine;
                                 use:
                                 jstat -J-Djstat.showUnsupported=true
                                 -snap <vmpid> | grep
                                 sun.management.JMXConnectorServer.address
                                 to find the url
 -port <mbean server port>       specify mbean server port, if missing -
                                 it will try to connect to MBean Server in
                                 the same VM
 -server <mbean server>          specify mbean server (localhost by
                                 default)
 -service <NameNode|DataNode>    specify jmx service (NameNode by default)
```